### DISCUSSION TITLE
맥북에서 갑자기 github.com에 접속되지 않습니다.

### DISCUSSION THREAD

**Discussion by nohjiho** [community/community#148008] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

맥북에서 깃허브에 접속이 잘되다 어느순간 부터 접속이 되고 있지 않습니다.
같은 경험을 하신 분이 계실까요?

맥북과 윈도우 노트북 두대를 쓰고 있는데. 같은 와이파이에 접속합니다.
윈도우 노트북에서는 접속이 되고 맥북에서는 접속이 되지 않아서 도움 요청합니다.

1. 크롬 브라우저 개발자 모드 네트워크 캡쳐
<img width="1678" alt="스크린샷 2024-12-27 오후 5 55 14" src="https://github.com/user-attachments/assets/3daa9db2-bebc-4710-bc17-3c0c006f7f7e" />

2. 아래는 맥북에서 https://github-debug.com/ 의  네트워크 접속 테스트 로그 입니다

1번 실행 결과
mac@macui-MacBookPro vivabutton_job_app % GIT_TRACE=1 GIT_TRANSFER_TRACE=1 GIT_CURL_VERBOSE=1 git clone https://github.com/github/debug-repo /tmp/debug-repo-http
17:18:54.185635 git.c:463               trace: built-in: git clone https://github.com/github/debug-repo /tmp/debug-repo-http
'/tmp/debug-repo-http'에 복제합니다...
17:18:54.204140 run-command.c:659       trace: run_command: git remote-https origin https://github.com/github/debug-repo
17:18:54.213891 git.c:749               trace: exec: git-remote-https origin https://github.com/github/debug-repo
17:18:54.214467 run-command.c:659       trace: run_command: git-remote-https origin https://github.com/github/debug-repo
17:18:54.232778 http.c:820              == Info: Couldn't find host [github.com](http://github.com/) in the .netrc file; using defaults
17:18:54.235241 http.c:820              == Info: Host [github.com:443](http://github.com:443/) was resolved.
17:18:54.235255 http.c:820              == Info: IPv6: (none)
17:18:54.235259 http.c:820              == Info: IPv4: 20.200.245.247
17:18:54.235287 http.c:820              == Info:   Trying 20.200.245.247:443...
17:20:09.237418 http.c:820              == Info: connect to 20.200.245.247 port 443 from 192.168.45.201 port 52074 failed: Operation timed out
17:20:09.237446 http.c:820              == Info: Failed to connect to [github.com](http://github.com/) port 443 after 75003 ms: Couldn't connect to server
17:20:09.237465 http.c:820              == Info: Closing connection
fatal: unable to access 'https://github.com/github/debug-repo/': Failed to connect to [github.com](http://github.com/) port 443 after 75003 ms: Couldn't connect to server
mac@macui-MacBookPro vivabutton_job_app % GIT_TRACE=1 GIT_TRANSFER_TRACE=1 GIT_CURL_VERBOSE=1 git clone git@github.com:github/debug-repo /tmp/debug-repo-ssh
17:20:50.427793 git.c:463               trace: built-in: git clone git@github.com:github/debug-repo /tmp/debug-repo-ssh
'/tmp/debug-repo-ssh'에 복제합니다...
17:20:50.440483 run-command.c:659       trace: run_command: unset GIT_DIR; GIT_PROTOCOL=version=2 ssh -o SendEnv=GIT_PROTOCOL [git@github.com](mailto:git@github.com) 'git-upload-pack '\''github/debug-repo'\'''
The authenticity of host '[github.com](http://github.com/) (20.200.245.247)' can't be established.
ED25519 key fingerprint is SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU.
This key is not known by any other names.
Are you sure you want to continue connecting (yes/no/[fingerprint])? ㅛㅛ yes
Warning: Permanently added '[github.com](http://github.com/)' (ED25519) to the list of known hosts.
[git@github.com](mailto:git@github.com): Permission denied (publickey).
fatal: 리모트 저장소에서 읽을 수 없습니다

올바른 접근 권한이 있는지, 그리고 저장소가 있는지
확인하십시오.
mac@macui-MacBookPro vivabutton_job_app % GIT_TRACE=1 GIT_TRANSFER_TRACE=1 GIT_CURL_VERBOSE=1 git clone git@github.com:github/debug-repo /tmp/debug-repo-ssh
17:21:24.391694 git.c:463               trace: built-in: git clone git@github.com:github/debug-repo /tmp/debug-repo-ssh
'/tmp/debug-repo-ssh'에 복제합니다...
17:21:24.403147 run-command.c:659       trace: run_command: unset GIT_DIR; GIT_PROTOCOL=version=2 ssh -o SendEnv=GIT_PROTOCOL [git@github.com](mailto:git@github.com) 'git-upload-pack '\''github/debug-repo'\'''
[git@github.com](mailto:git@github.com): Permission denied (publickey).
fatal: 리모트 저장소에서 읽을 수 없습니다

올바른 접근 권한이 있는지, 그리고 저장소가 있는지
확인하십시오.
mac@macui-MacBookPro vivabutton_job_app % ping -c 10 [github.com](http://github.com/)
PING [github.com](http://github.com/) (20.200.245.247): 56 data bytes
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=0 ttl=113 time=3.597 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=1 ttl=113 time=9.516 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=2 ttl=113 time=6.230 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=3 ttl=113 time=3.600 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=4 ttl=113 time=10.298 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=5 ttl=113 time=9.891 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=6 ttl=113 time=3.491 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=7 ttl=113 time=3.994 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=8 ttl=113 time=4.274 ms
64 bytes from [20.200.245.247](http://20.200.245.247/): icmp_seq=9 ttl=113 time=9.519 ms

--- [github.com](http://github.com/) ping statistics ---
10 packets transmitted, 10 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 3.491/6.441/10.298/2.851 ms
mac@macui-MacBookPro vivabutton_job_app % traceroute [github.com](http://github.com/)
traceroute to [github.com](http://github.com/) (20.200.245.247), 64 hops max, 40 byte packets
 1  192.168.45.1 (192.168.45.1)  2.189 ms  1.503 ms  1.472 ms
 2  * * *
 3  100.71.50.233 (100.71.50.233)  3.596 ms  2.858 ms  3.100 ms
 4  10.44.255.236 (10.44.255.236)  2.690 ms
    10.44.255.232 (10.44.255.232)  2.952 ms
    10.44.255.238 (10.44.255.238)  6.551 ms
 5  10.222.23.56 (10.222.23.56)  2.959 ms
    10.222.23.52 (10.222.23.52)  3.969 ms
    10.222.24.104 (10.222.24.104)  2.567 ms
 6  10.222.31.39 (10.222.31.39)  3.674 ms
    10.222.6.19 (10.222.6.19)  3.896 ms
    10.222.31.33 (10.222.31.33)  4.156 ms
 7  39.120.4.149 (39.120.4.149)  4.134 ms
    39.120.4.151 (39.120.4.151)  3.784 ms
    39.120.4.149 (39.120.4.149)  3.484 ms
 8  [ae29-0.icr02.sel21.ntwk.msn.net](http://ae29-0.icr02.sel21.ntwk.msn.net/) (104.44.239.244)  6.417 ms
    [ae22-0.icr01.sel21.ntwk.msn.net](http://ae22-0.icr01.sel21.ntwk.msn.net/) (104.44.42.135)  6.417 ms
    [ae29-0.icr02.sel21.ntwk.msn.net](http://ae29-0.icr02.sel21.ntwk.msn.net/) (104.44.239.244)  9.250 ms
 9  * * *
10  * * *
11  * * *
12  * * *
13  * * *
14  * * *
15  * * *
16  * * *
17  * * *
18  * * *
19  * * *
20  * * *
21  * * *
22  * * *
23  * * *
24  * * *
25  * * *
26  * * *
27  * * *
28  * * *
29  * * *
30  * * *
31  * * *
32  * * *
33  * * *
34  * * *
35  * * *
36  * * *
37  * * *
38  * * *
39  *^C
mac@macui-MacBookPro vivabutton_job_app % curl -s -o/dev/null -w "downloadspeed: %{speed_download} | dnslookup: %{time_namelookup} | connect: %{time_connect} | appconnect: %{time_appconnect} | pretransfer: %{time_pretransfer} | starttransfer: %{time_starttransfer} | total: %{time_total} | size: %{size_download}\n" https://github.com/
downloadspeed: 0 | dnslookup: 0.002202 | connect: 0.000000 | appconnect: 0.000000 | pretransfer: 0.000000 | starttransfer: 0.000000 | total: 75.003252 | size: 0

2번 실행 결과                
Error downloading from [github.com](http://github.com/)
2507998 bytes downloaded from [cloud.githubusercontent.com](http://cloud.githubusercontent.com/) at 8.97 Mbps
2507998 bytes downloaded from [avatars.githubusercontent.com](http://avatars.githubusercontent.com/) at 79.40 Mbps
2507998 bytes downloaded from [github.githubassets.com](http://github.githubassets.com/) at 13.98 Mbps
2507998 bytes downloaded from [australiaeast.github-debug.com](http://australiaeast.github-debug.com/) at 10.02 Mbps
2507998 bytes downloaded from [brazilsouth.github-debug.com](http://brazilsouth.github-debug.com/) at 4.74 Mbps
2507998 bytes downloaded from [centralindia.github-debug.com](http://centralindia.github-debug.com/) at 8.50 Mbps
2507998 bytes downloaded from [fra.github-debug.com](http://fra.github-debug.com/) at 7.31 Mbps
2507998 bytes downloaded from [iad.github-debug.com](http://iad.github-debug.com/) at 6.73 Mbps
2507998 bytes downloaded from [japaneast.github-debug.com](http://japaneast.github-debug.com/) at 27.69 Mbps
2507998 bytes downloaded from [koreacentral.github-debug.com](http://koreacentral.github-debug.com/) at 41.24 Mbps
2507998 bytes downloaded from [sea.github-debug.com](http://sea.github-debug.com/) at 10.92 Mbps
2507998 bytes downloaded from [southafricanorth.github-debug.com](http://southafricanorth.github-debug.com/) at 4.66 Mbps
2507998 bytes downloaded from [southeastasia.github-debug.com](http://southeastasia.github-debug.com/) at 17.67 Mbps
2507998 bytes downloaded from [github-cloud.s3.amazonaws.com](http://github-cloud.s3.amazonaws.com/) at 9.66 Mbps

                
58.123.36.147

                
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/[131.0.0.0](http://131.0.0.0/) Safari/537.36

                
e148c7d9092c2fa297bca12e77f75b69

                
C4AD:2D5677:7453:9EC4E:676E61A9

                
                  

                    
traceroute to 58.123.36.147 (58.123.36.147), 30 hops max, 60 byte packets
 1  *
 2  *
 3  *
 4  *
 5  *
 6  *
 7  *
 8  *
 9  *
10  *
11  *
12  *
13  *
14  *
15  *
16  *
17  *
18  *
19  *
20  *
21  *
22  *
23  *
24  *
25  *
26  *
27  *
28  *
29  *
30  *

                    
PING 58.123.36.147 (58.123.36.147) 56(84) bytes of data.

--- 58.123.36.147 ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2041ms

                    
Start: 2024-12-27T00:14:57-0800
HOST: 6181ca006a3ed04ddb24ba1f0584d689               Loss%   Snt   Last   Avg  Best  Wrst StDev

                    
Host 147.36.123.58.in-addr.arpa. not found: 3(NXDOMAIN)

미리 감사합니다.

**Comment by shinybrightstar** [comment]
Hi @nohjiho , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
